regression/pages/whitelabel/logout_page.py

```diff
@@ -27,5 +28,9 @@ def is_browser_on_page(self):
         """
         home_page = HomePage(self.browser)
 
-        return ("you have signed out" in self.browser.page_source.lower()) or \
-            home_page.is_browser_on_page()
+        try:
+            return ("you have signed out" in self.browser.page_source.lower()) or \
+                home_page.is_browser_on_page()
+        except WebDriverException:
```

Comment:
**Reviewer Notes**:
* First tried `TypeError`, but that failed with `WebDriverException`.
* Next tried `Exception`, which worked, but failed quality.
* Now trying `WebDriverException`.
If this last one doesn't pass microsites e2e, worst case, use `Exception` and disable quality error.

Reply:
`WebDriverException` seemed to do the trick, so I am resolving this.